src/Instamojo.php

```diff
@@ -320,18 +325,28 @@ public function paymentRequestPaymentStatus($id, $payment_id)
     * For more information on the allowed date formats check the
     * docs: https://www.instamojo.com/developers/request-a-payment-api/#toc-filtering-payment-requests
     */
-    public function paymentRequestsList($datetime_limits=null) 
+    public function paymentRequestsList( $limit = null , $page = null , $max_created_at = null , $min_created_at = null , $max_modified_at = null , $min_modified_at = null ) 
     {
         $endpoint = 'payment-requests';
 
-        if(!empty($datetime_limits)){
-            $query_string = http_build_query($datetime_limits);
+        $params = array();
+        if (!is_null($max_created_at)) {
+            $params['max_created_at'] = $max_created_at;
+        }
+
+        if (!is_null($min_created_at)) {
+            $params['min_created_at'] = $min_created_at;
+        }
 
-            if(!empty($query_string)){
-                $endpoint .= '/?' . $query_string;
-            }
+        if (!is_null($min_modified_at)) {
+            $params['min_modified_at'] = $min_modified_at;
         }
-        $response = $this->api_call('GET', $endpoint, array()); 
+
+        if (!is_null($$max_modified_at)) {
+            $params['max_modified_at'] = $max_modified_at;
+        }
+
+        $response = $this->api_call('GET', 'payment-requests', array_merge($params , $this->getParamsArray($limit , $page)));
         return $response['payment_requests'];
```

Comment:
This change might break the current integration, right?
Should we do it the old way only?

Reply:
I don't think so. We will have a new release version for this change.